Cargo.toml

```diff
@@ -20,6 +20,7 @@ rust-version = "1.75"
 exclude = ["org.freedesktop.Secrets.xml"]
 
 [workspace.dependencies]
+byteorder = "1.5.0"
```

Comment:
zbus itself uses https://crates.io/crates/endi, it would be nice if we can use that to avoid pulling the same "code" twice.

Reply:
Switched to using endi instead.